src/AWS.Deploy.CLI/Commands/CommandFactory.cs

```diff
@@ -32,6 +33,8 @@ public class CommandFactory : ICommandFactory
         private static readonly Option<bool> _optionDiagnosticLogging = new(new []{"-d", "--diagnostics"}, "Enable diagnostic output.");
         private static readonly Option<string> _optionApply = new("--apply", "Path to the deployment settings file to be applied.");
         private static readonly Option<bool> _optionDisableInteractive = new(new []{"-s", "--silent" }, "Disable interactivity to deploy without any prompts for user input.");
+        private static readonly Option<string> _optionOutputDirectory = new(new[]{"-o", "--output"}, "Directory path in which the CDK deployment project will be saved.");
+        private static readonly Option<string> _optionProjectDisplayName = new(new[] { "--project-display-name" }, "The name of the deployment project that will be displayed in the list of available deployment options.");
```

Comment:
Why do we need this?

Reply:
Refer to this comment - https://github.com/aws/aws-dotnet-deploy/pull/257#discussion_r671465771